site/_posts/2019-10-08-deactivating-reactivating-deleting-user-fusionauth-python.md

```diff
@@ -1,120 +1,129 @@
+For this tutorial I'm going to use [VENV](https://docs.python.org/3/library/venv.html), Python's virtual environment library, for app development and testing. If you have VENV installed in a folder called `venv` on your server like we did in the last tutorial, you can browse to the folder and start the virtual environment server with:
 
-```bash
-$ source venv/bin/activate
+```zsh
+source venv/bin/activate
 ```
 
 If you don't see `(venv)` prepended to your command prompt, your virtual environment is not running. It should look like this:
 
-```bash
+```zsh
 (venv) $
 ```
 
-If the FusionAuth-Client Python library is not already installed for venv, you can install it now with:
+If the FusionAuth Python client library is not already installed for venv, you can install it now with:
 
-```bash
-(venv) $ pip install fusionauth-client
+```zsh
+pip install fusionauth-client
 ```
 
 Change directories to your preferred test application directory, and let's get started!
 
-## Obtaining a Test User Account ID
+## Obtaining a test user account Id
 
-For this tutorial, we'll first need a test `user ID` that is registered with FusionAuth.
+For this tutorial, we'll first need a test user Id that is registered with FusionAuth.
 
 **IMPORTANT:** If your FusionAuth server only has one admin user listed, you should definitely use a separate test user account for this tutorial. Create a new one before you continue.
 
-In a web browser, let's go to **Users** in the FusionAuth UI. Select a test user, and under Actions, navigate to **Manage**, which should look like this.
+In a web browser, let's go to "Users" in the FusionAuth UI. Select a test user, and under Action, navigate to "Manage", which should look something like this.
 
-{% include _image.liquid src="/assets/img/blogs/action-manage-button.png" alt="Manage user" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/action-manage-button.png" alt="Manage user" class="img-fluid" figure=false %}
 
-On the **Manage User** page in the FusionAuth UI, you'll see the user ID located directly under the user's Email address. In this instance, it's `4e0f2af0-33a2-4001-8631-0f8225e65f1c`.
+On the "Manage User" page in the FusionAuth UI, you'll see the "User Id" located directly under the user's email address. In this instance, it's `4e0f2af0-33a2-4001-8631-0f8225e65f1c`.
 
 Copy it into a text editor and set it off to the side. We'll be using it in a moment and throughout the rest of the tutorial.
 
-{% include _image.liquid src="/assets/img/blogs/terry-test-account.png" alt="Terry test user info" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/terry-test-account.png" alt="Terry test user info" class="img-fluid" figure=false %}
 
-## How to Deactivate a User with FusionAuth-Client
+## Setting up FusionAuth using a Kickstart file
```

Comment:
I'd add a section here defining each of these terms. Something like this (but jazz it up if you want).

* deactivate, also known as a soft delete or account locking, lets you disable the ability for a user to login temporarily
* reactivate removes the lock and lets a user login
* delete, also known as a hard delete, removes all data for a user and can't be undone. Beware!

Reply:
looks good now.